dune_api_scripts/store_query_result_for_todays_trading_data.py

```diff
@@ -36,7 +36,7 @@ def build_query_for_todays_trading_volume():
     query_id = int(os.getenv("QUERY_ID_TODAYS_TRADING_DATA", "249240"))
     QUERY = build_query_for_todays_trading_volume()
     time_of_request = int(time.time())
-    dune_query = DuneQuery("", "", QUERY, Network.MAINNET, [], query_id)
+    dune_query = DuneQuery(query_id)
```

Comment:
I don't understand. How will the content of QUERY be passed into dune, if we don't hand it over here?

Reply:
OH! Yes... this was a typo (this query actually needs the query content). Good catch. Will fix.

Reply:
Fixed this @josojo - great catch!